src/ts/localization/localizationTemplate.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Since we have JSON files for English and German, the template should not be necessary?